notebooks/114-simplified-mode/114-simplified-mode.ipynb

```diff
@@ -0,0 +1,367 @@
+    "os.makedirs(CALIB_DIR, exist_ok=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prepare the calibration dataset\n",
+    "To prepare the calibration dataset we need to do the following:\n",
+    "- Download CIFAR10 dataset from Torchvision.datasets repository\n",
+    "- Save the selected number of elements from this dataset as .png images to the separate folder"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "transform = T.Compose([T.ToTensor()])\n",
+    "dataset = CIFAR10(root=CIFAR_DIR, train=False, transform=transform, download=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pil_converter = T.ToPILImage(mode=\"RGB\")\n",
+    "\n",
+    "for idx, info in enumerate(dataset):\n",
+    "    im = info[0]\n",
+    "    if idx >= CALIB_SET_SIZE:\n",
+    "        break\n",
+    "    label = info[1]\n",
+    "    pil_converter(im.squeeze(0)).save(Path(CALIB_DIR) / f'{label}_{idx}.png')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prepare the Model\n",
+    "Model preparation stage includes the following steps:,\n",
+    "- Download PyTorch model from Torchvision repository,\n",
+    "- Convert it to ONNX format,\n",
+    "- Run OpenVINO Model Optimizer tool to convert ONNX to OpenVINO Intermediate Representation (IR)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = torch.hub.load(\"chenyaofo/pytorch-cifar-models\", \"cifar10_resnet20\", pretrained=True)\n",
+    "dummy_input = torch.randn(1, 3, 32, 32)\n",
+    "\n",
+    "onnx_model_path = Path(MODEL_DIR) / 'resnet20.onnx'\n",
+    "ir_model_xml = onnx_model_path.with_suffix('.xml')\n",
+    "ir_model_bin = onnx_model_path.with_suffix('.bin')\n",
+    "\n",
+    "torch.onnx.export(model, dummy_input, onnx_model_path)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we convert this model into the OpenVINO IR using the Model Optimizer:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!mo --framework=onnx --data_type=FP32 --input_shape=[1,3,32,32] -m $onnx_model_path  --output_dir $MODEL_DIR"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Compression stage\n",
+    "Model compression stage includes the following steps:,\n",
+    "- Writing the optimization configuration and exporting it to .json file\n",
+    "- Calling the Post-Training Optimization Tool with the created configuration"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "config = dict({\n",
+    "    \"model\": {\n",
```

Comment:
Since there is nothing custom in the config for this model. Can we remove it and use POT purely from CLI so that the POT config will be generated on-fly. We started supporting this in 2022.1.